packages/bot-web-ui/src/components/quick-strategy/description-tab/martingale-description.tsx

```diff
@@ -0,0 +1,253 @@
+import { getImageLocation } from '../../../public-path';
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import '../quick-strategy.scss';
+
+export const MARTINGALE = [
+    {
+        title: <Localize key={0} i18n_default_text={'Exploring the Martingale strategy in Deriv Bot'} />,
+        content: (
+            <>
+                <div className='long_description__content'>
+                    <Localize
+                        key={0}
+                        i18n_default_text={
+                            'The Martingale strategy involves increasing your stake after each loss to recoup prior losses with a single successful trade.'
+                        }
```

Comment:
```suggestion
                        i18n_default_text='The Martingale strategy involves increasing your stake after each loss to recoup prior losses with a single successful trade.'
```

@farabi-deriv do we really need to wrap the text with the curly braces? this question applies to all of the other instances.

Reply:
@sandeep-deriv no its not needed. Already removed it.